tests/vmlifecycle_test.go

```diff
@@ -159,6 +159,32 @@ var _ = Describe("Vmlifecycle", func() {
 			})
 		})
 
+		Context("with boot order", func() {
+			It("should be able to boot from 'middle' disk", func() {
+				By("defining a VM with an Alpine disk")
+				vm = tests.NewRandomVMWithEphemeralDisk(tests.RegistryDiskFor(tests.RegistryDiskAlpine))
+				By("adding a Cirros Disk with boot order 1")
+				tests.AddEphemeralDiskWithBootOrder(vm, "disk1", "virtio", tests.RegistryDiskFor(tests.RegistryDiskCirros), 1)
+				By("adding a Fedora Disk")
+				tests.AddEphemeralDisk(vm, "disk2", "virtio", tests.RegistryDiskFor(tests.RegistryDiskFedora))
+
+				By("starting VM")
+				vm.Spec.Hostname = "cirros"
+				obj, err := virtClient.RestClient().Post().Resource("virtualmachines").Namespace(tests.NamespaceTestDefault).Body(vm).Do().Get()
+				Expect(err).To(BeNil())
+
+				By("waiting for VM to start")
+				_, ok := obj.(*v1.VirtualMachine)
+				Expect(ok).To(BeTrue(), "Object is not of type *v1.VM")
+				Expect(tests.WaitForSuccessfulVMStart(obj)).ToNot(BeEmpty())
```

Comment:
WaitForSuccessfulVMStart will time out and make the test fail. No need for the check. We should probably rename it to `ExpectSuccessfullVMStart` to make that clear.

Reply:
I can't seem to get the test to pass the Expect portions unless I include this to wait for the VM to start before proceeding.

Reply:
Yes yes, plese keep the `WaitForSuccessfulVMStart(obj)`. That is perfect. I simply meant that the wrapping `Expect()` is not needed.